Polish docs for the v1.1.0 release (#221)
harvester/docs: docs/install/iso-install.md

1. Choose the installation device on which the Harvester cluster will be installed
      - Note: By default, Harvester uses [GPT](https://en.wikipedia.org/wiki/GUID_Partition_Table) partitioning schema for both UEFI and BIOS. If you use the BIOS boot, then you will have the option to select [MBR](https://en.wikipedia.org/wiki/Master_boot_record).
   ![iso-install-disk.png](/img/v1.1/install/iso-install-disk.png)

> 1. Recommended choosing a separate disk for storing VM data.

Reviewer: ```suggestion
1. Recommended choosing a separate disk to store VM data.
```

   ![iso-install-disk.png](/img/v1.1/install/iso-select-data-disk.png )

> 1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster, in case the DHCP is used, the user must make sure the DHCP server always offers the same IP for the same Node. Due to a changed Node IP the related Node can not join the cluster, or even break the cluster)</small>.

Reviewer: ```suggestion
1. Configure the hostname and select the network interface for the management network. By default, Harvester will create a bonded NIC named `mgmt-bo`, and the IP address can be configured via DHCP or a statically assigned one <small>(Note: The Node IP can not change at the lifecycle of a Harvester cluster. If DHCP is used, users must make sure the DHCP server always offers the same IP for the same Node. If Node IP is changed, the related Node cannot join the cluster and might even break the cluster)</small>.
```

   ![iso-installed.png](/img/v1.1/install/iso-installed.png)
1. You will be prompted to set the password for the default `admin` user when logging in for the first time.
   ![first-login.png](/img/v1.1/install/first-time-login.png)

:::note

> In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation, see issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.

Reviewer: ```suggestion
In some cases, if you are using an older VGA connector, you may encounter an `panic: invalid dimensions` error with ISO installation. See issue [#2937](https://github.com/harvester/harvester/issues/2937#issuecomment-1278545927) for a workaround.
```
